packages/components/src/components/assay/RunDataPanel.tsx

```diff
@@ -52,6 +52,7 @@ interface Props {
     title: string
     fileSizeLimits?: Map<string, FileSizeLimitProps>
     maxInsertRows?: number
+    onDataChange?: (dirty: boolean, changeType: IMPORT_DATA_FORM_TYPES) => any
```

Comment:
Since there are other props which will be called for the other run data tabs (i.e. files or text blog changes) and this one is just called for editable grid row changes, should this be named more along those lines (i.e. onGridChange, onGridRowChange)?

Reply:
Good thought. I'll do that.